app/src/main/res/menu/login_options_menu.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/loginOptionsEditDelete"
+    android:contentDescription="@string/login_menu_more_options"
```

Comment:
I'm not sure we need to match this theme since this 3 dot menu will now look different from every other menu that we have in the app. Let's double check with UX about this.